.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release PECL Preparation
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  package:
+    name: Prepare package
+    runs-on: macos-latest
```

Comment:
curious again why we're using macos, if there's a reason, please add a comment explaining why